content/react/concepts/state/state.md

```diff
@@ -76,6 +76,41 @@ class Car extends React.Component {
 }
 ```
 
+## Using Property Initializers
+
+With Property Initializers you do not need to set state or bind methods in the constructor:
```

Comment:
Just wanted to point out that we like to refrain from using a "you"-voice in Docs entries.

Reply:
💯